ghcjs-src/Miso/Html/Internal.hs

```diff
@@ -263,6 +271,32 @@ onWithOptions options eventName Decoder{..} toAction =
    setProp "options" jsOptions eventHandlerObject
    setProp eventName eo (Object eventObj)
 
+-- | @onCreated toAction@ is an event that gets called after the actual DOM
+-- element is created. The @toAction@ is given a @sink :: action -> IO ()@ and
+-- the DOM element that was created. It can use those to embed third party
+-- widgets in the element. The @sink@ can be used to create callbacks for
+-- event listeners, and DOM element to attach the widget to.
+onCreated
+  :: ((action -> IO ()) -> Object -> action)
+  -> Attribute action
+onCreated toAction =
+  Attribute $ \sink n -> do
+    onCreatedArr <- SomeJSArray <$> getProp "onCreated" n
+    cb <- jsval <$> asyncCallback1 (sink . toAction sink . Object)
```

Comment:
@FPtje I see this callback (and also the `onDestroy` callback) is never released. This will cause a memory leak since all objects reachable from `toAction` and `sink` will be kept alive as long as the RTS is running.

The question is when to release the callback. Since the callbacks are only executed once we could just release them after they're called as in:

```javascript
for (var i in obj.onCreated){
  var cb = obj.onCreated[i];
  cb(obj.domRef);
  h$release(cb);
}
```

Would that work?

Reply:
Yes it would, I've delayed doing this to combine it with #363